landscaper/pkg/gardenlet/controller/gardenlet.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/gardener/component-spec/bindings-go/apis/v2"
```

Comment:
I known that there was a comment from @rfranzke about it but `make format` suggest a different formatting here (also visible in the verify step in the ci/cd):
```
$ make format
> Format
landscaper/pkg/gardenlet/controller/gardenlet.go
```

```diff
diff --git a/landscaper/pkg/gardenlet/controller/gardenlet.go b/landscaper/pkg/gardenlet/controller/gardenlet.go
index ee6e92132..878453b72 100644
--- a/landscaper/pkg/gardenlet/controller/gardenlet.go
+++ b/landscaper/pkg/gardenlet/controller/gardenlet.go
@@ -21,7 +21,7 @@ import (
        "io/ioutil"
        "time"

-       "github.com/gardener/component-spec/bindings-go/apis/v2"
+       v2 "github.com/gardener/component-spec/bindings-go/apis/v2"
        "github.com/gardener/component-spec/bindings-go/apis/v2/cdutils"
        "github.com/gardener/component-spec/bindings-go/codec"
        landscaperv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
```

Reply:
Yes that was it. thanks